src/components/Tabs/Tabs.less

```diff
@@ -95,7 +95,10 @@
 
 	&-Tab-is-active-and-open {
 		border-bottom: @Tabs-border-width solid @color-white;
-		z-index: 1; // Safari fix
+
+		// `z-index: -1` fixes a very minor visual bug here in IE 11, but it
```

Comment:
By removing the z-index here, it introduces a very minor bug in IE 11 that I'm willing to live with since this fixes other behavior on all the other browsers when `Tabs` are in `Dialogs`.